apps/researcher/src/messages/nl/messages.json

```diff
@@ -153,54 +153,54 @@
+  "leaveUndisturbed": "Onverstoord Laten",
+  "leaveUndisturbedLabel": "De _Onverstoord-Laten-notice_ wordt gebruikt om aan te geven dat items in een collectie niet mogen worden verstoord. Ze mogen alleen onder zeer beperkte omstandigheden, volgens gemeenschapsrichtlijnen, worden verplaatst of verstoord.",
   "safety": "Veiligheid",
-  "safetyLabel": "De Veiligheidskennisgeving wordt gebruikt om aan te geven dat uiterste voorzichtigheid en zorg nodig is bij het bekijken en behandelen van dit item of deze collectie voor het welzijn en de gezondheid van individuen of gemeenschappen.",
+  "safetyLabel": "De _Veiligheids-notice_ wordt gebruikt om aan te geven dat uiterste voorzichtigheid en zorg nodig is bij het bekijken en behandelen van dit item of deze collectie, voor het welzijn en de gezondheid van individuen of gemeenschappen.",
   "viewing": "Bekijken",
-  "viewingLabel": "De Bekijkkennisgeving wordt gebruikt om aan te geven dat er beperkingen zijn op wie, hoe en wanneer een item/collectie kan worden bekeken, gedistribueerd en gecirculeerd. Dit omvat bestaande afbeeldingen of representaties van een bestaand of gedeaccesioneerd item binnen een collectie.",
+  "viewingLabel": "De _Bekijken-notice_ wordt gebruikt om aan te geven dat er beperkingen zijn op wie, hoe en wanneer een item/collectie kan worden bekeken, gedistribueerd en gecirculeerd. Dit omvat bestaande afbeeldingen of representaties van een bestaand of gedeaccesioneerd item binnen een collectie.",
   "withholding": "Achterhouden",
-  "withholdingLabel": "De Achterhouden Kennisgeving wordt gebruikt om aan te geven dat er nooit reproducties van dit item of deze collectie mogen worden gemaakt. Als er al bestaande foto's of reproducties zijn, mogen deze niet worden gebruikt in een tentoonstelling of publicatie in print, online of in enig ander formaat."
+  "withholdingLabel": "De _Achterhouden-notice_ wordt gebruikt om aan te geven dat er nooit reproducties van dit item of deze collectie mogen worden gemaakt. Als er al bestaande foto's of reproducties zijn, mogen deze niet worden gebruikt in een tentoonstelling of publicatie in print, online of in enig ander formaat."
 },
 "LocalContextsNoticesOverview": {
-  "title": "Local Context Notices",
-  "description": "De Local Context Notices ondersteunen de opname van lokale protocollen voor toegang en gebruik van cultureel erfgoed dat digitaal circuleert buiten gemeenschapscontexten. <link> Meer informatie over de Lokale Context kennisgevingen.</link>.",
+  "title": "Local Contexts Notices",
+  "description": "De Local Contexts Collections Care (CC) Notices zijn zichtbare identificatiekenmerken die speciaal ontwikkeld zijn voor instellingen en depots om goede culturele zorg en beheer van collecties te ondersteunen, geleid en gestuurd door inheemse protocollen en autoriteit. <link> Meer informatie over de Local Contexts Collections Care Notices.</link>.",
```

Comment:
Misschien hier een opmerking maken over dem rechtsgeldigheid van de notices? Als er een licentie beperking bestaat dan **mag** data niet gebruik worden, als er een notice op rust, dan is het verzoek van de gemeenschap om het niet te gebruiken. Het is dan aan de gebruiker om daar gehoor aan te geven, notices kunnen niet gehandhaafd worden.

Reply:
denk je dat bovenstaande tekst de indruk wekt dat het hier niet om een verzoek gaat, maar om een plicht?